ht4/ht4.go

```diff
@@ -0,0 +1,31 @@
+package ht4
+
+func max(array []string) string {
```

Comment:
This is not the best name for func. eg `longestWord` would be more self explanatory

Reply:
This function has name "max" because this name was marked in description of the task